src/EditorFeatures/Core.Wpf/InheritanceMargin/InheritanceGlyphFactoryProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.InheritanceMargin
+{
+    [Export(typeof(IGlyphFactoryProvider))]
+    [Name(nameof(InheritanceGlyphFactoryProvider))]
+    [ContentType(ContentTypeNames.CSharpContentType)]
+    [ContentType(ContentTypeNames.VisualBasicContentType)]
+    [TagType(typeof(InheritanceMarginTag))]
+    [Order]
```

Comment:
what is the Order for?

Reply:
Some MEF parts require it. I'm not sure if this is one of those cases.

Order without constraints is explicit declaration that there are no ordering constraints.

Reply:
The comments for IGlyphFactoryProvider said it is required, but I am not sure where the image would be if there is no input. The order might need to be changed after the UI is checked in

Reply:
sgtm.